InternshipBE/BL/DTOs/FeedbackDTO.cs

```diff
@@ -0,0 +1,24 @@
+﻿using DAL.Entities;
+using Shared.Enums;
+using System;
+using System.Collections.Generic;
+
+namespace BL.DTOs
+{
+    public class FeedbackDTO
+    {
+        public int Id { get; set; }
+
+        public string UserId { get; set; }
+
+        public int CandidateId { get; set; }
+
+        public EnglishLevelType EnglishLevelType { get; set; }
+
+        public DateTime Date { get; set; }
+
+        public string Description { get; set; }
+
+        public ICollection<Evaluation> Evaluations { get; set; }
```

Comment:
?

Reply:
I just think about it and it's implementation. So i copy it from Entity model.